packages/static/kairos-overlay-files/files/etc/cos/bootargs.cfg

```
@@ -1,7 +1,8 @@
 function setSelinux {
     source (loop0)/etc/os-release
     set baseSelinuxCmd=""
-    if regexp "rockylinux|fedora|almalinux|redhat|tumbleweed" $KAIROS_ARTIFACT; then
+    # opensuse does not bundle regexp module so if we are on leap or tumbleweed we need to do this manually
+    if test $KAIROS_FLAVOR == "rockylinux" -o test $KAIROS_FLAVOR == "fedora" -o test $KAIROS_FLAVOR == "almalinux" -o test $KAIROS_FLAVOR == "redhat" -o test $ID == "opensuse-tumbleweed"; then
```

Can we use FAMILY instead?

```suggestion
    if test $KAIROS_FAMILY == "rhel" -o test $ID == "opensuse-tumbleweed"; then
```